packages/cfd/src/Components/cfd-real-account-display.tsx

```diff
@@ -95,12 +95,12 @@ const CFDRealAccountDisplay = ({
         const should_show_eu = (is_logged_in && is_eu) || (!is_logged_in && is_eu_country);
         const is_australian = residence === 'au';
         if (is_australian) {
-            return specifications[platform as keyof TSpecifications].au_real_financial_specs;
+            return specifications[platform as 'mt5'].au_real_financial_specs;
```

Comment:
@hamid-deriv If we are sure it's `mt5` at this point, Then we can just warp the code in an `if (platform === 'mt5')` to make TS happy and remove those type castings.

Reply:
You're right. As this file is out of this card's scope I didn't change the file more than this. BTW, I changed this.